### ISSUE TITLE
Model will not be uploaded unstar. 

### ISSUE THREAD

**Issue by Fibonacci134** [Seanseattle/StyleSwap#10]
Please remove this repo as you have no intention to deliver on your promise. Thanks. 

-The owner said they aren't releasing the model, no need to check up on the repo.

**Comment by instant-high** [comment]
There seem to be some connections to other similar projects on github, like HeSer (head swapper)
Same thing there, no code or models...

**Comment by netrunner-exe** [comment]
> There seem to be some connections to other similar projects on github, like HeSer (head swapper) Same thing there, no code or models...

Yes, one of the authors of this so-called paper is also one of the creators of HeSer. All the same - GitHub ignored, no evidence of project viability at all, and very high self-esteem. It seems to me that almost all the projects of these authors (except for MobileFaceSwap - but it's honestly hard to consider it a major project seriously) - just a scam. There has already been an attempt to make interpretation of the StyleSwap code according to their paper. Questions arose about their work - and as expected - the questions of the author who did the implementation were ignored.

**Comment by Yishaqel** [comment]
It was evident from the beginning that this would happen, I have removed the star and obviously I am disappointed, anyway this technology is advancing faster and faster and after seeing what is possible to do with Stable Diffusion probably better things will come out soon.

**Comment by netrunner-exe** [comment]
> It was evident from the beginning that this would happen, I have removed the star and obviously I am disappointed, anyway this technology is advancing faster and faster and after seeing what is possible to do with Stable Diffusion probably better things will come out soon.

I completely agree.  Well, the authors of this repository have now earned the fame of scammers.

**Comment by instant-high** [comment]
Ok.
Mobile faceswap is working quite good.
I made minor changes to it:
Rescale, crop output, keep audio...

Maybe they are not allowed to publish because model is trained using commercial data?
But nevertheless it's 'not amusing'...

**Comment by Yishaqel** [comment]
Yes, but there are also things about the paper that don't fit me too well in terms of "reliability" so they probably don't have such an effective model and if they do, it is not in our interests if it is not going to be published.

**Comment by netrunner-exe** [comment]
> Ok. Mobile faceswap is working quite good. I made minor changes to it: Rescale, crop output, keep audio...
> 
> Maybe they are not allowed to publish because model is trained using commercial data? But nevertheless it's 'not amusing'...

The model was trained on the VGGFace2 dataset, if I'm not mistaken.  It seems to me that the creation of this repository is just a formality or the usual hype around the topic of face swap, like HeSer and StyleSwap.  The initial goal was to publish a paper to make an achievement for conferences and portfolios, and to prove that it all works enough a video on YouTube.

**Comment by TaekyungKi** [comment]
For someone still interested in this work... here is some re-implementation codes
https://github.com/MingtaoGuo/StyleSwap

**Comment by netrunner-exe** [comment]
> For someone still interested in this work... here is some re-implementation codes https://github.com/MingtaoGuo/StyleSwap

MingtaoGuo had difficulties with training the model, as far as I know, he contacted the authors of this paper, but they did not answer him (as they do not give feedback everywhere). Therefore, perhaps that repository will not work correctly, unfortunately.
